cellxpredict/config.py

```diff
@@ -26,10 +26,7 @@ class ConfigBase:
     layers: List[int] = field(default_factory=lambda: [8, 16, 32, 64])
 
     def filename(self, component: str = "weights"):
-        filename = (
-            f"{self.model}_{component}_capacity={self.capacity}_"
-            f"z={self.latent_dims}_gamma={self.gamma}"
-        )
+        filename = (f"{self.model}_{component}")
```

Comment:
Also, I simplified the `config.filename()` method to output simpler name of the saved weights, e.g. `encoder_weights.h5` - we have the `ConfigHyperParams.json` file which logs all the attributes now!